### PR TITLE
docs: optimal-cluster-size -> 2.0/admin_guide

### PR DESCRIPTION
**Pull request by yichengq**
/cc @robszumski

### CODE REVIEW

**Comment by robszumski** [comment]
Can we move over this entire section? https://github.com/coreos/etcd/blob/master/Documentation/optimal-cluster-size.md#odd-active-cluster-size

File: Documentation/2.0/admin_guide.md [hunk 1]
```diff
@@ -27,11 +27,34 @@ The data directory has two sub-directories in it:
...
 If you are spinning up multiple clusters for testing it is recommended that you specify a unique initial-cluster-token for the different clusters.
 This can protect you from cluster corruption in case of mis-configuration because two members started with different cluster tokens will refuse members from each other.
 
+#### Optimal Cluster Size
+
+The recommended etcd cluster size is 3, 5 or 7, which is decided by the fault tolerance requirement. A 7-member cluster can provide enough fault tolerance in most cases. While larger cluster provides better fault tolerance, its write performance becomes lower since data needs to be replicated to more machines.
+
+#### Fault Tolerance Table
+
+The recommended cluster size is odd number. Adding an odd member to the cluster doesn't change the size of the majority, so you gain a higher tolerance for failure by adding the extra member. You can see this in practice when comparing two even and odd sized clusters:
```
Comment (robszumski) [inline Documentation/2.0/admin_guide.md]: > comparing two even and odd sized clusters

Can you remove "two" from this

Comment (robszumski) [inline Documentation/2.0/admin_guide.md]: It is recommended to have an odd number of members in a cluster. Having an odd cluster size doesn't change the number needed for majority, but you gain a higher tolerance for failure by adding the extra member. You can see this in practice when comparing even and odd sized clusters:

Comment (yichengq) [inline Documentation/2.0/admin_guide.md]: thx. updated.

**Comment by robszumski** [comment]
LGTM

**Pull request merged**